fetch-in-react/src/app.jsx

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-console */
+import React from 'react';
+import UserList from './user-list';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      users: [],
+      isLoading: true
+    };
+  }
+
+  componentDidMount() {
+    /* your code here */
+    fetch('https://jsonplaceholder.typicode.com/users')
+      .then(res => res.json())
+      .then(data => console.log(data))
+      .catch(error => { console.error('Error:', error); });
```

Comment:
The state of the app should get updated so that it renders the list of users when they're loaded.